app/assets/stylesheets/_typography.scss

```diff
@@ -90,6 +90,7 @@ p {
   font-size: $font-size-xs;
   font-weight: $font-weight-light;
   line-height: 1.38;
+  word-break: break-word;
```

Comment:
The value `break-word` doesn't exist for `word-break`. See the [doc](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break).